examples/literate/D_simulatingFloatingPlatforms.jl

```diff
@@ -0,0 +1,394 @@
+    1 "M6" 4 2 ptfm_mass[4, 2]
+    1 "M6" 4 3 ptfm_mass[4, 3]
+    1 "M6" 4 4 ptfm_mass[4, 4]
+    1 "M6" 4 5 ptfm_mass[4, 5]
+    1 "M6" 4 6 ptfm_mass[4, 6]
+    1 "M6" 5 1 ptfm_mass[5, 1]
+    1 "M6" 5 2 ptfm_mass[5, 2]
+    1 "M6" 5 3 ptfm_mass[5, 3]
+    1 "M6" 5 4 ptfm_mass[5, 4]
+    1 "M6" 5 5 ptfm_mass[5, 5]
+    1 "M6" 5 6 ptfm_mass[5, 6]
+    1 "M6" 6 1 ptfm_mass[6, 1]
+    1 "M6" 6 2 ptfm_mass[6, 2]
+    1 "M6" 6 3 ptfm_mass[6, 3]
+    1 "M6" 6 4 ptfm_mass[6, 4]
+    1 "M6" 6 5 ptfm_mass[6, 5]
+    1 "M6" 6 6 ptfm_mass[6, 6]
+]
+bottomConcTerms = OWENSFEA.applyConcentratedTerms(
+    bottomMesh.numNodes,
+    numDOFPerNode,
+    data=bottomConcInputs,
+    jointData=[])
+bottomFEAModel = OWENS.FEAModel(;
+    analysisType = structuralModel,
+    outFilename = "none",
+    joint = [],
+    platformTurbineConnectionNodeNumber = 1,
+    nlOn = true,
+    numNodes = bottomMesh.numNodes,
+    RayleighAlpha = 0.05,
+    RayleighBeta = 0.05,
+    iterationType = "DI",
+    gamma = 1.0,
+    alpha = 0.5,
+    nodalTerms=bottomConcTerms)
+
+nothing
+
+# Finally, some additional structure definitions for the bottom mesh.
+# These don't contain anything new, but are required for OWENS to run properly.
+bottomEl = OWENSFEA.El(bottomSectionProps,
+                       bottomOrt.Length,
+                       bottomOrt.Psi_d,
+                       bottomOrt.Theta_d,
+                       bottomOrt.Twist_d,
+                       ones(bottomMesh.numEl))
+
+nothing
+
+# We can now run our floating simulation in OWENS.
+# Note that we are using `OWENS.Unsteady` now instead of `OWENS.Unsteady_Land`.
+
+println("Running Unsteady")
+t, aziHist, OmegaHist, OmegaDotHist, gbHist, gbDotHist, gbDotDotHist, FReactionHist, FTwrBsHist,
+genTorque, genPower, torqueDriveShaft, uHist, uHist_prp,
+epsilon_x_hist, epsilon_y_hist, epsilon_z_hist, kappa_x_hist, kappa_y_hist, kappa_z_hist,
+FPtfmHist, FHydroHist, FMooringHist = OWENS.Unsteady(inputs,
+    system=topSystem,
+    assembly=topAssembly,
+    topModel=topModel,
+    topMesh=topMesh,
+    topEl=topEl,
+    aero=aeroForces,
+    deformAero=deformAero,
+    bottomModel=bottomModel,
+    bottomMesh=bottomFEAMesh,
+    bottomEl=bottomEl,
+    bin=bin)
+
+nothing
+
+# Since we ran the simulation for such a short time period for sake of demonstrating
+# completion of the simulation, the plots below are not too interesting.
+# Try experimenting with changing the `numTS` variable above to a longer simulation to
+# see how the floating VAWT responds over time.
+# Note that the platform modeled here is not necessarily design for the Darrieus VAWT it
+# is supporting, so your mileage may vary on the simulation results.
+
+PyPlot.plot(t, uHist_prp[:, 1])
+PyPlot.title("Platform Surge Response")
+PyPlot.xlabel("Time [s]")
+PyPlot.ylabel("Force [N]")
+
+PyPlot.plot(t, FHydroHist[:, 1])
+PyPlot.xlabel("Time [s]")
+PyPlot.ylabel("Force [N]")
+PyPlot.title("Hydrodynamic Surge Loading")
+
+nothing
```

Comment:
I think this is all great, if we'd be able to visualize the resulting hydrodyn and moordyn and topside vtk outputs with paraview, and drop a screenshot of that here, that might be really cool.